sources/packages/web/src/views/supporting-user/AppSupportingUser.vue

```diff
@@ -6,7 +6,12 @@
         @click="goToDashboard"
       ></BCLogo>
       <v-spacer></v-spacer>
-      <v-btn v-if="isAuthenticated" class="mr-5" text @click="goToDashboard"
+      <v-btn
+        v-if="isAuthenticated"
+        class="mr-5"
+        text
```

Comment:
is this working, i think i need to use `variant="text"`

Reply:
That's working:
![image](https://user-images.githubusercontent.com/78114138/195691203-2c5c805a-8968-4078-8463-87b44d5c4336.png)